kornia/geometry/conversions.py

```diff
@@ -374,19 +373,16 @@ def rotation_matrix_to_angle_axis(rotation_matrix: Tensor) -> Tensor:
 
 
 def rotation_matrix_to_quaternion(
-    rotation_matrix: Tensor, eps: float = 1.0e-8, order: QuaternionCoeffOrder = QuaternionCoeffOrder.XYZW
+    rotation_matrix: Tensor, eps: float = 1.0e-8, order: QuaternionCoeffOrder = QuaternionCoeffOrder.WXYZ
```

Comment:
shouldn't we remove the `order` parameter everywhere and the [enum](https://github.com/kornia/kornia/blob/master/kornia/geometry/conversions.py#L58) itself, since the `xyzw` is removed. CC\ @edgarriba

Reply:
Yes, otherwise could be confusing that does nothing c @pri1311